From ae26631edf4aabcd44ca3f71d09e2619d7dd65cf Mon Sep 17 00:00:00 2001
From: rugnathr <rugnath.ram@intel.com>
Date: Thu, 25 Jul 2019 20:01:00 +0530
Subject: [PATCH] Fix error for unused private field

Moving unused private field to public
error: private field 'tcc_offset_checked' is not used [-Werror,-Wunused-private-field]
error: private field 'tcc_offset_low' is not used [-Werror,-Wunused-private-field]
---
 src/thd_engine_default.h | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/src/thd_engine_default.h b/src/thd_engine_default.h
index b6a5d10..b2833d4 100644
--- a/src/thd_engine_default.h
+++ b/src/thd_engine_default.h
@@ -38,10 +38,11 @@ private:
 
 	cthd_cpu_default_binding def_binding;
 	int workaround_interval;
-	int tcc_offset_checked;
-	int tcc_offset_low;
 
 public:
+        int tcc_offset_checked;
+        int tcc_offset_low;
+
 	static const int power_clamp_reduction_percent = 5;
 
 	cthd_engine_default() :
-- 
2.22.0

